src/pages/Cart/Cart.jsx

```diff
@@ -0,0 +1,181 @@
+import axios from "axios";
+import { useEffect, useReducer, useState } from "react/cjs/react.development";
+
+import { useAuth } from "../../context/Authentication";
+import { useCart } from "../../context/CartContext";
+import * as All from "../../assets/icons/icons.jsx";
+import "./cart.css"
+import { cartFn } from "./cartFn";
+// import { removeItemFromCartHandler } from "../../backend/controllers/CartController";
+
+
+ export function Cart(){
+  
+    const [btnDisable,setBtnDisable]=useState(false);
+     const {cart,setCart}=useCart();
+     const {login}=useAuth();
+     const [state,dispatch]=useReducer(cartFn,{items:[],qty:1,subTotal:0});
+    
+    
+    const removeHandler=(productid)=>{
+        (async ()=>{
+            try {
+                
+                const token=localStorage.getItem("login");
+                const res= await axios({method:"DELETE",url:`/api/user/cart/${productid}`,headers:{authorization:token}});
+                setCart(res.data.cart);
+                console.log("response from delete",res.data.cart);
+                dispatch({type:"INCREASE",payload:res.data.cart!==undefined?res.data.cart:[]});
+                
+            } catch (error) {
+                console.error(error);
+            }
+        })()
+    }
+
+    const increaseHandler=(productid)=>{
+        
+        (async ()=>{
+            try {
+                setBtnDisable(true);
+                const token=localStorage.getItem("login");
+               const res= await axios({method:"POST",url:`/api/user/cart/${productid}`,data:{action:{type:"increment"}},headers:{authorization:token}});
+               setCart(res.data.cart);
+               console.log("response",res.data.cart);
+               dispatch({type:"INCREASE",payload:res.data.cart!==undefined?res.data.cart:[],initialQty:res.data.cart[0].qty});
+               
+               setBtnDisable(false);
+            } catch (error) {
+                console.error(error);
+            }
```

Comment:
So this error gives us an object inside of which we have a message you can destructure and used to show the alert button to the user.